hack/jenkins/e2e-image/Dockerfile

```diff
@@ -25,10 +25,10 @@ ENV E2E_UP=true \
 
 # Customize these as appropriate
 ENV E2E_PUBLISH_GREEN_VERSION=false \
-    GINKGO_PARALLEL_NODES=25 \
-    GINKGO_TEST_ARGS="--ginkgo.focus=\[Conformance\]" \
     INSTANCE_PREFIX=jenkins-e2e \
     KUBERNETES_PROVIDER=gce
+    #GINKGO_PARALLEL_NODES=25 \
```

Comment:
maybe add a comment explaining why these are commented out / what the plan is to uncomment?